repo/docs/External_Services.md

```diff
@@ -234,6 +234,12 @@ global:
       arkcase:
         url: "ldaps://ldap:636"
         domain: "my.external-ldap.domain.com"
+        # enableCreatingLdapUsers, enableEditingLdapUsers and syncEnabled default to true
+        # If disabled there will be no connection between LDAP users and Arkcase users,
+        # which is preferential when SSO is used and there are a lot of Arkcase users
+        enableCreatingLdapUsers: "true" 
+        enableEditingLdapUsers: "true"
+        syncEnabled: "true"
```

Comment:
This being the case, shouldn't all these flags be hardcoded to `false` when SSO is in play?

Reply:
Hmmm.... well not necessarily... OWC is the only project where we will surely have issues because we would need to create thousands of users on LDAP. For any other future project we can evaluate the need to turn LDAP user creation off.

Reply:
I would instead argue the opposite: we shouldn't be synchronizing LDAP users unless:

- they have logged into the system
- they have been directly and expressly referenced as a participant (or such) within the system

If neither of the above is true, there's no need for the user to exist in the system.

If we're concerned about the user listings, then that has a different solution: download the LDAP users into _*memory*_ (or a temporary list somewhere) and serve it up from there. Thus, we only "create" things when they're needed, and not a second before.

Which is the opposite of what we've been doing historically.